libraries/redshop/src/Cart/Cart.php

```diff
@@ -226,6 +226,7 @@ public static function add($post = [])
         if (true !== $result) {
             return $result;
         }
+        $cart = self::modify($cart);
```

Comment:
please help me double check again if it's necessary to call function modify cart here

Reply:
I checked, it not necessary to call function modify cart, it work fine for me. Please check again, thank you.